package.json

```diff
@@ -1,8 +1,5 @@
 {
-  "name": "@dream/team",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "name": "@dream-team/monorepo",
```

Comment:
name full shit

Reply:
Agree with @AmateurBoy.
We can probably name it like "dream-project" or "blue cherry"